index.js

```diff
@@ -47,11 +61,23 @@ getUsersCreditDetails(usersArray);
 // ***************************************************************************
 // Iteration 4 - practice `.filter()` method and how to return two elements
 // ***************************************************************************
-
 const genderView = users => {
-  // Your code goes here ...
+  const femaleUsers = [];
+  const maleUsers = [];
+  const femaleUsersTest = users.filter((user) => {
+    return user.gender === 'female';
+  });
+  const maleUsersTest = users.filter((user) => {
+    return user.gender === 'male';
+  });
+  for (let user of femaleUsersTest){
+    femaleUsers.push(`${user.firstName} ${user.lastName}`)
+  };
+  for (let user of maleUsersTest){
+    maleUsers.push(`${user.firstName} ${user.lastName}`)
+  };
+  return `femaleUsers: ${femaleUsers}, maleUsers: ${maleUsers}`
```

Comment:
Here you are trying to send an array as a string. In some cases, Javascript might throw an error. Sometimes it just stringifies an array, but in this case, the goal could have been to return an object with the data:
```js
return {
    femaleUsers,
    maleUsers
}
```
This way you return an object with the two arrays